docs/GUIDE.md

```diff
@@ -0,0 +1,379 @@
+# Add Realtime 3D Avatars to Live Video Streams
+
+In today's rapidly evolving digital landscape, live-stream video is dominating real-time communication. Users now expect more immersive and customizable streaming options. Content creators are increasingly seeking creative new ways to stream themselves, giving rise to the demand for dynamic 3D avatars that mirror their movements and expressions.
+
+![](/docs/images/Intro-Demo.gif "Real-Time Avatars with Agora")
+
+Real-time virtual avatars traditionally required complex motion capture equipment and sophisticated software, often making them inaccessible to everyday users and independent creators. However, artificial intelligence has changed this status quo as well. With advancements in computer vision, it's now possible to run sophisticated AI algorithms on-device that can accurately capture and translate human facial gestures into digital form in real-time.
+
+In this walkthrough, we'll look at how to integrate 3D virtual avatars into your [Agora](https://www.agora.io) live streams using [MediaPipe](https://ai.google.dev/edge/mediapipe/solutions/guide) and 3D avatars from [ReadyPlayerMe](https://readyplayer.me/). Whether you're looking to enhance audience engagement or just add a fun, creative twist to your app's video calls/live broadcasts, this guide will provide you with the necessary steps to bring 3D virtual personas to life.
+
+## Prerequisites 
+- [Node.JS](https://nodejs.org)
+- A developer account with [Agora](https://console.agora.io)
+- A basic understanding of HTML/CSS/JS
+- A basic understanding of ThreeJS
+- A basic understanding of Agora - [Web QuickStart](https://medium.com/agora-io/a-simple-approach-to-building-a-group-video-chat-web-app-0b8a6cacfdfd)
+- A code editor, I use [VSCode](https://code.visualstudio.com)
+- A 3D avatar from [ReadyPlayerMe](https://readyplayer.me/)
+
+## Agora + MediaPipe project
+To keep this guide concise, I assume you understand how to implement the Agora Video SDK into a web app; if you don't, check out my guide on [ Building a Group Video Chat Web App](https://medium.com/agora-io/a-simple-approach-to-building-a-group-video-chat-web-app-0b8a6cacfdfd).
+
+To get started, download the [demo project](https://github.com/digitallysavvy/agora-mediapipe-readyplayerme). With the code downloaded, navigate to the project folder in the terminal and use `npm` to install the node packages.
+
+```bash
+git clone git@github.com:digitallysavvy/agora-mediapipe-readyplayerme.git
+cd agora-mediapipe-readyplayerme
+npm i 
+```
+
+## Core Structure (HTML) 
+Let’s start with the HTML structure in [`index.html`](index.html), at the top of the `<body>` are the "call" UI elements. This includes a container for the remote videos, a container for the local user with buttons for muting and unmuting the audio/video, and a button to leave the chat.
```

Comment:
"This includes" should be "These include". 

Adding in the inferred subject "elements" makes it easier to spot.

Reply:
updated